src/pages/dar_application/DataAccessRequestApplication.jsx

```diff
@@ -505,6 +534,13 @@ const DataAccessRequestApplication = (props) => {
   const { dataRequestId } = props.match.params;
   const eRACommonsDestination = isNil(dataRequestId) ? 'dar_application' : ('dar_application/' + dataRequestId);
 
+  if (isLoading) {
+    return <SpinnerComponent
+        show={true}
+        name="loadingSpinner"
+        loadingImage={loadingImage}
+    />;
+  }
```

Comment:
Add a loading spinner for the page, rather than just in the DucAddendum component

Reply:
Hoping to revisit loading in a follow on ticket because the loading spinner flashes on/off b/c we're repeatedly hitting the same endpoints multiple times. So, we should take another look at the useEffects in this component.